requirements.txt

```diff
@@ -0,0 +1,236 @@
+anyio==3.1.0
+appdirs==1.4.4
+argon2-cffi==20.1.0
+asgiref==3.4.1
+astunparse==1.6.3
+async-generator==1.10
+async-timeout==3.0.1
+attrs==21.2.0
+audioread==2.1.9
+Babel==2.9.1
+backcall==0.2.0
+beautifulsoup4==4.10.0
+bleach==3.3.0
+blessings==1.7
+blis==0.7.4
+botocore==1.23.14
+CacheControl==0.12.6
+cachetools==4.2.2
+Camelot==12.6.29
+catalogue==2.0.6
+certifi==2020.12.5
+cffi==1.14.5
+chardet==4.0.0
+charset-normalizer==2.0.6
+clang==5.0
+click==7.1.2
+colorama==0.4.4
+comtypes==1.1.10
+cycler==0.10.0
+cymem==2.0.5
+Cython==0.29.23
+dataframe-image==0.1.1
+decorator==5.0.9
+defusedxml==0.7.1
+Deprecated==1.2.13
+distlib==0.3.1
+Django==3.2.5
+dnspython==2.1.0
+docx2txt==0.8
+Elixir==0.7.1
+email-validator==1.1.2
+en-core-web-sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.1.0/en_core_web_sm-3.1.0-py3-none-any.whl
+entrypoints==0.3
+filelock==3.0.12
+firebase-admin==5.0.2
+Flask==1.1.2
+Flask-Login==0.5.0
+Flask-Migrate==2.7.0
+Flask-SQLAlchemy==2.4.4
+Flask-WTF==0.14.3
+flatbuffers==1.12
+gast==0.4.0
+gcloud==0.18.3
+gensim==3.8.2
+geographiclib==1.52
+geopy==2.2.0
+google-api-core==1.31.0
+google-api-python-client==2.14.0
+google-auth==1.33.1
+google-auth-httplib2==0.1.0
+google-auth-oauthlib==0.4.6
+google-cloud-core==2.0.0
+google-cloud-firestore==2.3.2
+google-cloud-storage==1.42.1
+google-crc32c==1.1.2
+google-pasta==0.2.0
+google-resumable-media==2.0.2
+googleapis-common-protos==1.53.0
+gpustat==0.6.0
+grpcio==1.40.0
+gunicorn==20.0.4
+h5py==3.1.0
+httplib2==0.19.1
+idna==2.10
+ipykernel==5.5.5
+ipython==7.23.1
+ipython-genutils==0.2.0
+itsdangerous==2.0.1
+jedi==0.18.0
+Jinja2==3.0.2
+jmespath==0.10.0
+joblib==1.0.1
+json5==0.9.5
+jsonify==0.5
+jsonschema==3.2.0
+jupyter-client==6.1.12
+jupyter-core==4.7.1
+jupyter-server==1.8.0
+jupyterlab==3.0.16
+jupyterlab-pygments==0.1.2
+jupyterlab-server==2.5.2
+jws==0.1.3
+keras==2.6.0
+Keras-Preprocessing==1.1.2
+keyboard==0.13.5
+kiwisolver==1.3.1
+librosa==0.8.1
+llvmlite==0.37.0
+lxml==4.6.3
+Mako==1.1.5
+Markdown==3.3.4
+MarkupSafe==2.0.1
+matplotlib==3.4.2
+matplotlib-inline==0.1.2
+mistune==0.8.4
+monotonic==1.6
+mpld3==0.5.5
+msgpack==1.0.2
+multidict==5.2.0
+murmurhash==1.0.5
+nbclassic==0.3.1
+nbclient==0.5.3
+nbconvert==6.0.7
+nbformat==5.1.3
+nest-asyncio==1.5.1
+nltk==3.6.2
+notebook==6.4.0
+numba==0.54.0
+numpy==1.19.5
+nvidia-ml-py3==7.352.0
+oauth2client==4.1.3
+oauthlib==3.1.1
+opencensus==0.8.0
+opencensus-context==0.1.2
+opencv-contrib-python==4.4.0.46
+opt-einsum==3.3.0
+packaging==20.9
+pandas==1.1.5
+pandocfilters==1.4.3
+parso==0.8.2
+pathy==0.6.0
+pbr==5.6.0
+pdf2image==1.16.0
+pickleshare==0.7.5
+Pillow==8.2.0
+playsound==1.2.2
+pooch==1.5.1
+preshed==3.0.5
+prometheus-client==0.10.1
+prompt-toolkit==3.0.18
+proto-plus==1.19.0
+protobuf==3.17.3
+psutil==5.8.0
+py-spy==0.3.11
+pyasn1==0.4.8
+pyasn1-modules==0.2.8
+PyAudio @ file:///C:/Users/User/AppData/Local/Programs/Python/Python39/Scripts/PyAudio-0.2.11-cp39-cp39-win_amd64.whl
```

Comment:
I don't think PyAudio is really a thing that should exist in the `requirements.txt` file.